Source/WebService/Blackjack.WebService/ClientApp/src/app/components/play-game/play-game.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="mainBody">
+  <div class="fundRelated">
```

Comment:
Subjective personal preference here. "fundRelated" sounds weird to me. I would go with something like, "balanceAndWagerContainer" but the code works. Just lending my perspective